plat/arm/common/arm_bl2_setup.c

```diff
@@ -44,12 +44,6 @@ CASSERT(BL2_BASE >= (ARM_BL_RAM_BASE + BL1_MEMINFO_OFFSET), assert_bl2_base_over
 #pragma weak bl2_plat_arch_setup
 #pragma weak bl2_plat_sec_mem_layout
 
-#if LOAD_IMAGE_V2
-
-#pragma weak bl2_plat_handle_post_image_load
-
-#else /* LOAD_IMAGE_V2 */
-
```

Comment:
You've now got a mismatched #if/#endif. You want this to become `#if !LOAD_IMAGE_V2` so that all the code to line 171 is removed when LOAD_IMAGE_V2 is true.